Gemfile.lock

```diff
@@ -231,5 +231,8 @@ DEPENDENCIES
   web-console (>= 3.3.0)
   webpacker
 
+RUBY VERSION
+   ruby 2.4.0p0
```

Comment:
Just to be clear here, the Ruby Version was included in this file when I ran `bundle install` so because we specify the version in the `ruby_version` and gem file. Not sure why it wasn't there before. Maybe its a RVM vs RBENV. Not sure.